Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/survey/SurveyQuestions.kt

```diff
@@ -164,28 +160,37 @@ private fun SingleChoiceQuestion(
                 Unit
             }
             val optionSelected = text == selectedOption
-            Row(
-                modifier = Modifier
-                    .fillMaxWidth()
-                    .selectable(
-                        selected = optionSelected,
-                        onClick = onClickHandle
-                    )
-                    .padding(vertical = 4.dp),
-                verticalAlignment = Alignment.CenterVertically
+            Surface(
+                shape = MaterialTheme.shapes.small,
+                border = BorderStroke(
+                    width = 1.dp,
+                    color = MaterialTheme.colors.onSurface.copy(alpha = 0.12f)
+                ),
+                modifier = Modifier.padding(vertical = 8.dp)
             ) {
-                RadioButton(
-                    selected = optionSelected,
-                    onClick = onClickHandle,
-                    colors = RadioButtonConstants.defaultColors(
-                        selectedColor = MaterialTheme.colors.primary
+                Row(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .selectable(
+                            selected = optionSelected,
+                            onClick = onClickHandle
+                        )
+                        .padding(vertical = 16.dp, horizontal = 24.dp),
+                    verticalAlignment = Alignment.CenterVertically,
+                    horizontalArrangement = Arrangement.SpaceBetween
+                ) {
+                    Text(
+                        text = text
                     )
-                )
 
-                Text(
-                    text = text,
-                    modifier = Modifier.padding(horizontal = 16.dp)
-                )
+                    RadioButton(
+                        selected = optionSelected,
+                        onClick = onClickHandle,
```

Comment:
As both the `Row` and `RadioButton` are clickable do you get separate ripples? I think it will; what about having a single ripple by hoisting `remember { InteractionState() }` and passing this to both the `clickable` and the `RadioButton`?

Reply:
this isn't possible right now: [see issue](https://issuetracker.google.com/171819073)

Reply:
ack